src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K, V>[] nodes;
+    private int size;
+    private double threshold;
+
+    public MyHashMap() {
+        nodes = new Node[DEFAULT_CAPACITY];
+        threshold = DEFAULT_CAPACITY * LOAD_FACTOR;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size > threshold) {
+            resize();
+        }
+        int index = getIndex(key);
```

Comment:
```suggestion
        int index = getIndex(key) % nodes.length;
```
because now you have method what gives you hash